infra/main.bicep

```diff
@@ -108,6 +108,9 @@ module openAi2 'core/ai/cognitiveservices.bicep' = {
 module apim 'core/gateway/apim.bicep' = {
   name: 'apim'
   scope: resourceGroup
+  dependsOn: [
```

Comment:
should not be needed, only the child API created on this APIM requires the dependencies